bin/deploy

```diff
@@ -33,15 +33,15 @@ test -f "$LSSTSW/miniconda/.deployed" || ( # Anaconda
 
     miniconda_file_name="Miniconda-${MINICONDA_VERSION}-${ana_platform}.sh"
     echo "::: Deploying Miniconda ${MINICONDA_VERSION} for ${ana_platform}"
-    curl -# -L -O http://repo.continuum.io/miniconda/${miniconda_file_name}
-    bash ${miniconda_file_name} -b -p "$LSSTSW/miniconda"
+    curl -# -L -O "http://repo.continuum.io/miniconda/${miniconda_file_name}"
+    bash "$miniconda_file_name" -b -p "${LSSTSW}/miniconda"
 
     if [[ $(uname -s) = Darwin* ]]; then
```

Comment:
I thought we got rid of this -- I think GalSim can now build w/o this hack (at least on pre-El Capitan versions).

The problem is that this may make a build succeed on `lsstsw`, where it would fail with vanilla Anaconda. Maybe open a new PR?